### DISCUSSION TITLE
Deploy to Vercel, cannot require Postgress integration

### DISCUSSION THREAD

**Discussion by Ben2W** [vercel/community#4516] [Feature Question] [answered]
### Question

[So when you configure a "Deploy to Vercel" button in the documentation](https://vercel.com/docs/deployments/deploy-button), Vercel supports requiring an integration. Is there a way to require a Vercel Postgress integration? It appears the Deploy to Vercel button doesn't have a way to require that.

**Comment by amyegan** [comment, marked as answer]
You're right, the button configuration page doesn't have Storage options yet. But you can include Vercel Postgres by adding the query parameter `stores=[{"type":"postgres"}]` to your button URL.

**Reply by Ben2W** [reply]
That worked, thanks!
